osu.Game/Online/API/Requests/Responses/APIChangelogEntry.cs

```diff
@@ -48,6 +48,7 @@ public class APIChangelogEntry
     public enum ChangelogEntryType
     {
         Add,
-        Fix
+        Fix,
+        Misc
```

Comment:
sort of a drive-by change, but i'll look away for that one

Reply:
This is actually a valid type, just haven't used it in years.

Reply:
Yeah I checked web, just saying that it's not hugely related to the main purpose of the PR.